src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -317,6 +343,7 @@ describe('ReactTestRenderer', () => {
     ]);
   });
 
+  // this is only passing in Fiber because refs aren't actually working
```

Comment:
Is this comment outdated? It’s not telling me much: are refs broken in Fiber? Are they broken specifically for test renderer? What is our plan to fix them? Is this within the scope of this PR?